activiti-cloud-runtime-bundle-service/activiti-cloud-services-runtime-bundle/activiti-cloud-services-core/src/main/java/org/activiti/cloud/services/core/BaseProcessDefinitionService.java

```diff
@@ -52,4 +54,10 @@ protected ExtendedCloudProcessDefinition decorate(
             .map(decorator -> decorator.decorate(processDefinition))
             .orElse(processDefinition);
     }
+
+    protected GetProcessDefinitionsPayload buildGetProcessDefinitionsPayload(boolean latestVersions) {
+        var processDefinitionsPayloadBuilder = ProcessPayloadBuilder.processDefinitions();
+        processDefinitionsPayloadBuilder.withLatestVersionOnly(latestVersions);
+        return processDefinitionsPayloadBuilder.build();
+    }
```

Comment:
This method is used in a single place and I think it's not really needed, you can inline it directly chaining the methods on builder: `ProcessPayloadBuilder.processDefinitions().withLatestVersionOnly(latestVersions).build()`